src/Symfony/Component/Config/Builder/ConfigBuilderGenerator.php

```diff
@@ -413,39 +413,43 @@ private function getComment(BaseNode $node): string
     {
         $comment = '';
         if ('' !== $info = (string) $node->getInfo()) {
-            $comment .= ' * '.$info."\n";
+            $comment .= $info."\n";
         }
 
         if (!$node instanceof ArrayNode) {
             foreach ((array) ($node->getExample() ?? []) as $example) {
-                $comment .= ' * @example '.$example."\n";
+                $comment .= '@example '.$example."\n";
             }
 
             if ('' !== $default = $node->getDefaultValue()) {
-                $comment .= ' * @default '.(null === $default ? 'null' : var_export($default, true))."\n";
+                $comment .= '@default '.(null === $default ? 'null' : var_export($default, true))."\n";
             }
 
             if ($node instanceof EnumNode) {
-                $comment .= sprintf(' * @param ParamConfigurator|%s $value', implode('|', array_unique(array_map(fn ($a) => !$a instanceof \UnitEnum ? var_export($a, true) : '\\'.ltrim(var_export($a, true), '\\'), $node->getValues()))))."\n";
+                $comment .= sprintf('@param ParamConfigurator|%s $value', implode('|', array_unique(array_map(fn ($a) => !$a instanceof \UnitEnum ? var_export($a, true) : '\\'.ltrim(var_export($a, true), '\\'), $node->getValues()))))."\n";
```

Comment:
When up-merging into 7.2, a backslash is added before `sprintf` by https://github.com/symfony/symfony/commit/6ce530c5e90397d88e3a76a56db266c74d651584#diff-2aaa94a5e0cdde514853eddaf724fb3724c0501ddc5285af115bc1fe9c04d7ce